src/main/scala/it/pagopa/interop/backendforfrontend/api/impl/EServicesApiServiceImpl.scala

```diff
@@ -263,9 +263,10 @@ final case class EServicesApiServiceImpl(
     val result = for {
       producerId    <- getOrganizationIdFutureUUID(contexts)
       consumerUUIDs <- Future.traverse(parseArrayParameters(consumersIds))(_.toFutureUUID)
-      eServicesIds  <-
-        if (consumerUUIDs.isEmpty) Future.successful(Nil)
-        else getProducerEServicesIds(producerId, consumerUUIDs)
+      eServicesIds  <- getProducerEServicesIds(producerId, consumerUUIDs)
+      _             <-
+        if (consumerUUIDs.isEmpty || eServicesIds.isEmpty) Future.successful(Nil)
+        else Future(())
```

Comment:
this code is not changing the behaviour because it is always successful and the value is always discarded

Reply:
+1